# Completely empty dataset. We expect it to fail by throwing exception.
- query: >-
    CALL link_prediction.train()
    YIELD training_results, validation_results
    RETURN training_results, validation_results;
  exception: "Empty dataset. "
